dhis-2/dhis-api/src/test/java/org/hisp/dhis/textpattern/TestTextPatternMethod.java

```diff
@@ -38,8 +38,9 @@
     @Test
     void testValidateText()
     {
-        String[] valid = { "\"Hello world!\"", "\"Hello \\\"world\\\"\"" };
-        String[] invalid = { "Hello world", "Hello \" world", "\"Hello world", "Hello world\"" };
+        String[] valid = { "\"Hello world!\"", "\"Hello \\\"world\\\"\"",
+            "\"This is a text with more than two words\"" };
+        String[] invalid = { "Hello world", "Hello \" world", "\"Hello world", "Hello world\"", "\"Hello \"world\"\"" };
```

Comment:
I think this test should be replaced by a parametrized test so each case can fail independently.